nbbuild/javadoctools/export2html.xsl

```diff
@@ -80,10 +80,13 @@
         <em>NetBeans</em> one is free to choose the set of modules and their APIs 
         to satisfy one's needs.
         <p>
-        This is a list of APIs for development version, if you want to see
+        This is a list of APIs for Apache NetBeans (incubating) 10.0 version, if you want to see
         a list of APIs for a particular version, you may want to go to:
         </p>
         <ul>
+            <li><a href="http://bits.netbeans.org/8.1/javadoc/" target="_top">Dev</a> - Javadoc as released for developement version</li>
```

Comment:
it should use dev instead of a number but this will for a moment link to (Oracle NetBeans dev). 
AFAIK
: bits.netbeans.org will be hosted at the nebeans-vm and older link will point to former bits.netbeans.org at oracle